src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
Don't begin class implementation with an empty line. Remove the redundant empty line at the start of the class definition.